src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1239,6 +1239,35 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="Make_0_inheritable" xml:space="preserve">
     <value>Make '{0}' inheritable</value>
   </data>
+  <data name="Apply_file_header_preferences" xml:space="preserve">
+    <value>Apply file header preferences</value>
+  </data>
+  <data name="Apply_object_collection_initialization_preferences" xml:space="preserve">
+    <value>Apply object collection initialization preferences</value>
+  </data>
+  <data name="Apply_this_qualification_preferences" xml:space="preserve">
+    <value>Apply Me qualification preferences</value>
+  </data>
+  <data name="Apply_using_directive_placement_preferences" xml:space="preserve">
+    <value>Apply Import directive placement preferences</value>
+    <comment>{Locked="Import"} "Import" is a VB keyword and should not be localized.</comment>
```

Comment:
value does not match data.

Reply:
The data name matches what is in editor.Config which is common with C#, the Value is what should appear in the Option configuration for VB Code Cleanup.

Reply:
> The data name matches what is in editor.Config

these are not editorconfig key names.  these are resource string names.  resource string names should match the resource string value.

Reply:
Fixed